src/commands/login.ts

```diff
@@ -137,8 +140,21 @@ export const loginCommand = (yargs: yargs.Argv) =>
           type: "boolean",
           describe: "Force re-authentication",
           default: false,
+        })
+        .option("debug", {
+          type: "boolean",
+          describe: "Print debug information.",
         }),
-    fsShutdownGuard(login)
+    fsShutdownGuard(
+      (
+        args: yargs.ArgumentsCamelCase<{
+          org: string;
+          refresh?: boolean;
+          debug?: boolean;
+        }>
+      ) =>
+        login({ org: args.org, refresh: args.refresh }, { debug: args.debug })
```

Comment:
nit: could we pass in login(args, args)?

Reply:
🤔 We could... it's shorter for sure.